src/components/Layout/index.tsx

```diff
@@ -11,6 +11,7 @@ import NetworkId from './NetworkId';
 import SocialAndTermLinks from './SocialAndTermLinks';
 import Versions from './Versions';
 import './styles.sass';
+import ChainSeletor from '../ChainSelector';
```

Comment:
Nit: Typo

Reply:
jeeez, you can call it whatever you like when is the default haha. fixed